node/src/providers/ecdsa/sign.rs

```diff
@@ -45,12 +46,25 @@ impl EcdsaSignatureProvider {
                 .ok_or_else(|| anyhow::anyhow!("Payload is not an ECDSA payload"))?,
             tweak: sign_request.tweak,
             entropy: sign_request.entropy,
-        }
-        .perform_leader_centric_computation(
-            channel,
-            Duration::from_secs(self.config.signature.timeout_sec),
-        )
-        .await?;
+        };
+        let res = sign_computation
+            .perform_leader_centric_computation(
+                channel,
+                Duration::from_secs(self.config.signature.timeout_sec),
+            )
+            .await;
+
+        let (signature, public_key) = match res {
+            Ok(res) => res,
+            e => {
+                participants.iter().for_each(|id| {
+                    metrics::PARTICIPANT_TOTAL_TIMES_SEEN_IN_FAILED_COMPUTATION
+                        .with_label_values(&[&id.raw().to_string()])
+                        .inc();
+                });
+                return e;
+            }
+        };
```

Comment:
Any reason we only track this for ECDSA signatures when we are the leader? I'd assume it would be nice to track computations we're participating in as follower as well, and also EDDSA signatures. Or was this intentional for any reason?

Reply:
Thanks!